libhif/hif-types.h

```diff
@@ -82,6 +83,7 @@ typedef enum {
         HIF_ERROR_REPO_NOT_FOUND                = 19,   /* Since: 0.1.0 */
         HIF_ERROR_FAILED_CONFIG_PARSING         = 24,   /* Since: 0.1.0 */
         HIF_ERROR_PACKAGE_NOT_FOUND             = 8,    /* Since: 0.1.0 */
+        HIF_ERROR_NO_SPACE                      = 46,   /* Since: 0.2.3 */
```

Comment:
hm, 0.2.3? Shouldn't it be 0.7.0? Or we want to put this into 0.2.3 release as well?